### PR TITLE
Update db schema comments

### PR DESCRIPTION
**Pull request by hryvinskyi**
### Description
Update DB Schema Comments

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @hryvinskyi. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/CatalogRule/etc/db_schema.xml [hunk 1]
```diff
@@ -156,7 +157,8 @@
     </table>
     <table name="catalogrule_customer_group" resource="default" engine="innodb"
            comment="Catalog Rules To Customer Groups Relations">
-        <column xsi:type="int" name="rule_id" padding="10" unsigned="true" nullable="false" identity="false"/>
+        <column xsi:type="int" name="rule_id" padding="10" unsigned="true" nullable="false" identity="false"
+                comment="Rule Id"/>
```
Comment (rogyar) [inline app/code/Magento/CatalogRule/etc/db_schema.xml]: Should it be `Rule ID` as well? :)

Comment (hryvinskyi) [reply app/code/Magento/CatalogRule/etc/db_schema.xml]: yes, it's a typo :)

**Comment by magento-engcom-team** [comment]
Hi @hryvinskyi. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**